blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/ModelBuilder.java

```diff
@@ -44,12 +44,26 @@
     /*package*/ final Map<Class, Model> models = new ConcurrentHashMap<Class, Model>();
 
     public <T> Model<T> get(Class<T> type) throws NotExportableException {
+        if (type.getAnnotation(ExportedBean.class) == null) {
```

Comment:
Directly updating stapler imported code? This can leave us with pain later on, better to submit PR on stapler then we import them over here.